libs/iavl/mutable_tree.go

```diff
@@ -409,25 +409,29 @@ func (tree *MutableTree) LoadVersion(targetVersion int64) (int64, error) {
 	latestVersion := int64(0)
 
 	var latestRoot []byte
-	for version, r := range roots {
-		tree.versions.Set(version, true)
-		if version > latestVersion && (targetVersion == 0 || version <= targetVersion) {
-			latestVersion = version
-			latestRoot = r
-		}
-		if firstVersion == 0 || version < firstVersion {
-			firstVersion = version
+	if targetVersion != 20 {
+		for version, r := range roots {
+			tree.versions.Set(version, true)
+			if version > latestVersion && (targetVersion == 0 || version <= targetVersion) {
+				latestVersion = version
+				latestRoot = r
+			}
+			if firstVersion == 0 || version < firstVersion {
+				firstVersion = version
+			}
 		}
-	}
 
-	if !(targetVersion == 0 || latestVersion == targetVersion) {
-		return latestVersion, fmt.Errorf("wanted to load target %v but only found up to %v",
-			targetVersion, latestVersion)
-	}
+		if !(targetVersion == 0 || latestVersion == targetVersion) {
+			return latestVersion, fmt.Errorf("wanted to load target %v but only found up to %v",
+				targetVersion, latestVersion)
+		}
 
-	if firstVersion > 0 && firstVersion < int64(tree.ndb.opts.InitialVersion) {
-		return latestVersion, fmt.Errorf("initial version set to %v, but found earlier version %v",
-			tree.ndb.opts.InitialVersion, firstVersion)
+		if firstVersion > 0 && firstVersion < int64(tree.ndb.opts.InitialVersion) {
+			return latestVersion, fmt.Errorf("initial version set to %v, but found earlier version %v",
+				tree.ndb.opts.InitialVersion, firstVersion)
+		}
+	} else {
+		latestVersion = 20
```

Comment:
why set latestVersion to 20, what's the meaning of 20 in this context

Reply:
fixed .it's venus1Height